systori/templates/project/project_list.html

```diff
@@ -94,12 +94,12 @@
                 <button class="glyphicon glyphicon-remove" id="search_clear" class="hidden"></button>
             </div>
             <div class="btn-group">
-                <sys-sort-button type="button" data-type="id" data-reversed=true class="btn btn-xs btn-default">
-                    {% trans "ID ↓↑" %}</sys-sort-button>
-                <sys-sort-button type="button" data-type="name" data-reversed=false class="btn btn-xs btn-default">
-                    {% trans "name ↓↑" %}</sys-sort-button>
-                <sys-sort-button type="button" data-type="phase" data-reversed=true class="btn btn-xs btn-default active">
-                    {% trans "phase ↓↑" %}</sys-sort-button>
+                <button is="sys-sort-button" type="button" data-type="id" data-reversed=true class="btn btn-xs btn-default">
```

Comment:
What does this `is` attribute do?

Reply:
it changes the button element to a custom element which extends button. but I've already switched back to extends HTMLElement - with that we can have a custom tag <sys-sort-button></sys-sort-button> again, which is nicer.